web/src/main/java/com/epam/rft/atsy/web/mapper/RuleValidationExceptionMapper.java

```diff
@@ -1,8 +1,20 @@
 package com.epam.rft.atsy.web.mapper;
 
+/**
```

Comment:
Follow Javadoc conventions: write the first sentence as a short summary of the calss, as Javadoc automatically places it in the class summary table (and index).